akka-http-core/src/main/scala/akka/http/impl/engine/client/pool/NewHostConnectionPool.scala

```diff
@@ -301,7 +300,7 @@ private[client] object NewHostConnectionPool {
           }
           def isClosed: Boolean = requestOut.isClosed || responseIn.isClosed
 
-          def onPush(): Unit = {
+          def onPush(): Unit = try {
```

Comment:
What does `try` achieve here?

Reply:
Oops, looks like a left-over. Weird that it compiles.